agent/agents/mysql/perfschema/perfschema.go

```diff
@@ -55,22 +54,9 @@ type mySQLVersion struct {
 
 // versionsCache provides cached access to MySQL version.
 type versionsCache struct {
```

Comment:
It looks like we can remove `versionCache` at all

Reply:
@artemgavrilov 
Thanks reviewing!
Do you mean that `versionCache` itself should remove from `PerfSchema` struct member?

Reply:
Yes, as I see it's unused.